angularjs-portal-frame/src/main/webapp/css/buckyless/home.less

```diff
@@ -216,7 +216,7 @@ div.table-cell {
 .list-content .card-info,
 .list-content .card-remove,
 .widget-content .card-info,
-.widget-content .card-remove {
+.widget-content .card-remove, {
```

Comment:
Ah right, i was trying something then backed it out. Looks like I missed a comma :) . #fixing.